JOSESwift/Sources/RSAKeyCodable.swift

```diff
@@ -62,7 +62,7 @@ extension RSAPublicKey: Decodable {
 
         // Other common parameters are optional.
         var parameters: [String: String] = [:]
-        for key in commonParameters.allKeys {
+        for key in commonParameters.allKeys where key != .X509CertificateChain {
```

Comment:
Please also ignore the `keyOperations` parameter, [it is an array of strings](https://tools.ietf.org/html/rfc7517#section-4.3) as well.

```suggestion
        for key in commonParameters.allKeys where key != .X509CertificateChain && key != .keyOperations {
```

Reply:
It would be awesome if you could also add the workaround to the decoding of `RSAPrivateKey` [here](https://github.com/xaverlohmueller/JOSESwift/blob/96fe42f8a12ed790b2637c870d5671b602a07af4/JOSESwift/Sources/RSAKeyCodable.swift#L127).